### PR TITLE
Update 06-KeithWinovich-eaton.md

### PR DESCRIPTION
**Pull request by KeithWinovich-eaton**

### CODE REVIEW

File: _slides/06-KeithWinovich-eaton.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to our slide deck! keith changed the file again"
```
Comment (cody-carlson-learning) [inline _slides/06-KeithWinovich-eaton.md]: Hey what if we considered something else.

**Review by cody-carlson-learning — CHANGES_REQUESTED** [review]
How about this?

File: _slides/06-KeithWinovich-eaton.md (hunk 1 above)
Comment (cody-carlson-learning) [reply _slides/06-KeithWinovich-eaton.md]: LGTM!

**Review by cody-carlson-learning — APPROVED** [review]
Looks great, let's send it!

File: _slides/06-KeithWinovich-eaton.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to our slide deck! keith changed the file again!"
```
Comment (cody-carlson) [inline _slides/06-KeithWinovich-eaton.md]: Let's start a review

File: _slides/06-KeithWinovich-eaton.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
```
Comment (cody-carlson) [inline _slides/06-KeithWinovich-eaton.md:2]: Hey this is another comment

File: _slides/06-KeithWinovich-eaton.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
-title: "Welcome to our slide deck!"
+title: "Welcome to our slide deck! keith changed the file again!!!!"
```
Comment (cody-carlson) [inline _slides/06-KeithWinovich-eaton.md:3]: yep more comments

**Review by cody-carlson — CHANGES_REQUESTED** [review]
hey can we review?

**Review by cody-carlson — APPROVED** [review]

**Pull request merged**